src/ServiceControl/ExternalIntegrations/CustomCheckFailedPublisher.cs

```diff
@@ -1,51 +1,21 @@
 namespace ServiceControl.ExternalIntegrations
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using Raven.Client;
     using CustomCheckFailed = ServiceControl.Contracts.CustomCheckFailed;
 
-
-    public class CustomCheckFailedPublisher : EventPublisher<Contracts.CustomChecks.CustomCheckFailed, CustomCheckFailedPublisher.DispatchContext>
+    public class CustomCheckFailedPublisher : EventPublisher<Contracts.CustomChecks.CustomCheckFailed, CustomCheckFailed>
     {
-        protected override DispatchContext CreateDispatchRequest(Contracts.CustomChecks.CustomCheckFailed @event)
+        protected override CustomCheckFailed Convert(Contracts.CustomChecks.CustomCheckFailed message)
```

Comment:
This is good because it will catch existing instances that are in transit during an update. Would it make sense to go back a step and just publish the original external integration event at the point that we published the event being published here? Or should that happen in a future update?